selfdrive/car/gm/interface.py

```diff
@@ -219,11 +219,16 @@ def _get_params(ret, candidate, fingerprint, car_fw, experimental_long, docs):
       CarInterfaceBase.configure_torque_tune(candidate, ret.lateralTuning)
 
     elif candidate == CAR.SILVERADO:
-      ret.mass = 2200. + STD_CARGO_KG
+      ret.mass = 2500. + STD_CARGO_KG
       ret.wheelbase = 3.75
       ret.steerRatio = 16.3
       ret.centerToFront = ret.wheelbase * 0.5
       tire_stiffness_factor = 1.0
+      ret.steerActuatorDelay = 0.5
+      ret.vEgoStopping = 0.1
+      ret.stopAccel = -0.25
+      ret.longitudinalTuning.kpV = [2.4, 1.2]
+      ret.longitudinalTuning.kiV = [0.16]
```

Comment:
I would make sure to match the stock system first, if possible. Watch a few stock routes coming to a stop (or provide some)